### PR TITLE
Updates sqlalchemy-utc

### PR DESCRIPTION
**Pull request by lucasvbleite**
Sets it to sqlalchemy-utc>=0.14

DTWIN-981

job https://eden.esss.co/jenkins/job/all4well-fb-DTWIN-981-update-sqlalchemy-utc-alfatwin-newlinux/1/consoleFull

### CODE REVIEW

**Review by brnomendes — APPROVED** [review]

**Review by brnomendes — COMMENTED** [review]
shouldn't these PRs be merged into the rc?

File: environment.devenv.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ dependencies:
   {% endif %}
   - sqlalchemy ==1.3.15  # [PY==36]
   - sqlalchemy ==1.4.44  # [PY==310]
-  - sqlalchemy-utc>=0.10
+  - sqlalchemy-utc>=0.14
```
Comment (lincrosenbach) [inline environment.devenv.yml:22]: Should we restrict the use of older versions on serialchemy? @lvoliveira

**Review by lincrosenbach — CHANGES_REQUESTED** [review]
Maybe not add the restriction here

File: environment.devenv.yml (hunk 1 above)
Comment (lvoliveira) [reply environment.devenv.yml:22]: Only if serialchemy requires a feature that is only available on newer versions, and would brake with older ones. Is that the case? If yes, this is only one place where that change should be done, we would also need to change the `setup.py` file.

Comment (lucasvbleite) [reply environment.devenv.yml:22]: Not really, I will close this PR then

**Pull request closed without merging**